src/main/Server.java

```diff
@@ -0,0 +1,31 @@
+import clientHandler.ClientHandlerThread;
```

Comment:
Add short but clear JavaDoc documentations to *all* of your public classes and to *most* of your methods and private classes (not really needed for trivial methods like getters and setters). Will make parsing your code much easier. See the [javadocs tutorial.](http://www.oracle.com/technetwork/articles/java/index-137868.html). Author mentions in classes are fine, but please no auto-generated author mentions.